people.md

```diff
@@ -227,6 +227,22 @@ title: People
     </div>
   </div>  
 
+  <div class="card d-block">
+    <img class="card-img-top" src="https://avatars2.githubusercontent.com/u/24504585?s=400&amp;u=c8217c41b2f168f006676b8b0a5c2580b5d427ff&amp" alt="Mehdi Keshani">
+    <div class="card-body">
+      <div class="card-title">
+        <a href="https://ashkboos.github.io/MyWebsite/">Mehdi Keshani</a>
+      </div>
+      <p class="card-text">Software analytics, Software Evolution</p>
+    </div>
+    <div class="card-footer bg-transparent border-success">
+      <a href="https://twitter.com/Mehdi22043845" title="Twitter"><i class="fab fa-twitter"></i></a>
```

Comment:
The Twitter URL handle is invalid, fix it (or remove it) and then the PR should look good

Reply:
fixed!